powerpc/build.sh

```diff
@@ -10,43 +10,61 @@ set -e
 alias python=python3
 
 # NOTE tmp_dir is the prefix (bin, lib, include, build)
-#tmp_dir=$working_directory/tmp
-# TODO mario
 tmp_dir=$env_prefix
```

Comment:
We should check the number of arguments, and error out if no arguments are given